src/openforms/forms/tests/admin/test_form_definition.py

```diff
@@ -126,3 +126,27 @@ def test_deleted_forms_not_in_used_in(self):
             deleted_form_url,
             str(response.content),
         )
+
+    def test_cannot_mark_form_definition_as_non_reusable_if_used_in_multiple_forms(
+        self,
+    ):
+        step = FormStepFactory.create(form_definition__is_reusable=True)
+        FormStepFactory.create(form_definition=step.form_definition)
+
+        self.client.force_login(user=self.user)
+
+        response = self.app.get(
```

Comment:
this `self.client.force_login` does nothing, because `self.client` and `self.app` have nothing to do with each other.